docs/qa/CometBFT-QA-34.md

```diff
@@ -1,12 +1,14 @@
 ---
 order: 1
 parent:
```

Comment:
What does this parent section actually do?

Reply:
this is just the way the navigation works, to make the QA pages first level entries under the QA nav section